docs/viplab3.0/computation_result.md

```diff
@@ -1,141 +1,235 @@
+
+|Key            |Value Type |Opt / Must |Description |Comment |
+|---------------|-----------|-----------|------------|--------|
+| stdout        | string    | must      | the stdout of the exection, base64url encode | must be provided even if empty |
+| stderr        | string    | must      | the stderr of the exection, base64url encode | must be provided even if empty | 
+
+
+### artifact JSON objects
+
+The base artifact object that needs to be extended all other artifacts defined.
+
+#### artifact JSON object
+|Key            |Value Type |Opt / Must |Description |Comment |
+|---------------|-----------|-----------|------------|--------|
+| identifier    | string (uuid) |must   | unique id for this artifact | |
+| type          |  string   | must      | the type of this artifact | The type may be used by the client to determine how to display the artifact| 
+
+
+#### file artifact JSON object
+|Key            |Value Type |Opt / Must |Description |Comment |
+|---------------|-----------|-----------|------------|--------|
+| path          | string    | must      | the path in which the content had been created during the computation | |
+| MIMEtype      | string    | must      | MIMEtype of the conten | | 
+| content       | string    | must      | base64url encoded content of the file | | 
+
+In Viplab 2.0, the following MIMEtypes where allowed. They contain some custom defined types, which need to be keep supported:
+
+* text/plain
+* text/html
+* text/uri-list": links to websides to be presented in SC
+* image/png"
+* application/x-vgf: generated graphics output in 'ViP graphics format'
+* application/x-vgf3: generated graphics output in 'ViP graphics format 3D'
+* application/x-vgfc: generated graphics output in 'ViP graphics format contour plot'.|
+
+
+#### s3file artifact JSON object
+|Key            |Value Type |Opt / Must |Description |Comment |
+|---------------|-----------|-----------|------------|--------|
+| path          | string    | must      | the path in which the content had been created during the computation | |
+| MIMEtype      | string    | must      | MIMEtype of the conten | | 
+| url           | string    | must      | url on which the content of the file can be retrieved | | 
+| size          | int       | must      | size in bytes of the file stored on the external system | |
+
+
+#### notifications artifact JSON object
+Each notifications artifact contains at least one summary and unlimited number of notification elemets providing more details
+
+|Key            |Value Type |Opt / Must |Description |Comment | AS |
+|---------------|-----------|-----------|------------|--------|----|
+| summary | string | must | summary of all info_element's: e.g. one warning, one error may have a summary "An error occured." | E.g.: "[Error] Backend has detected an error: no result!" or "Success!" |
+| elements |  array of notification objects | opt | the more detailed notification objects for this notifications object| |
```

Comment:
`elements` is too generic we should rename it

Reply:
I did replace it with notifications. It makes it clear what is expected in this list